docs/RenderersGuide.md

```diff
@@ -0,0 +1,366 @@
+---
+authors:
+  - "@carson-katri"
```

Comment:
```suggestion
  - [@carson-katri](https://github.com/carson-katri)
```

Reply:
It's just a frontmatter key, which isn't displayed in the rendered document. I'll move it out of the frontmatter block to render properly.

Reply:
Oh ok that's fine then.